optaplanner-core/src/test/java/org/optaplanner/core/impl/localsearch/DefaultLocalSearchPhaseTest.java

```diff
@@ -213,4 +215,15 @@ public void solveListVariable() {
         assertThat(solution).isNotNull();
     }
 
+    @Test
+    public void solveListVariableWithExternalizedInverseAndIndexSupplies() {
+        SolverConfig solverConfig = PlannerTestUtils.buildSolverConfig(
+                TestdataListSolutionExternalized.class, TestdataListEntityExternalized.class);
+
+        TestdataListSolutionExternalized solution = TestdataListSolutionExternalized.generateUninitializedSolution(6, 2);
+
+        solution = PlannerTestUtils.solve(solverConfig, solution);
+        assertThat(solution).isNotNull();
+    }
```

Comment:
Maybe this test and the one above deserve a better place (a separate test class) but where? I can't find a suitable existing package for that.

Reply:
Adding it here is consistent with what we've done so far.

But if this is the best place for all those methods here, that is indeed not sure. Maybe there's a better structure that would be easier to find them, good for another PR maybe. I am not too worried about this now.